nodejs/sqlcommenter-nodejs/packages/sqlcommenter-sequelize/index.js

```diff
@@ -12,29 +12,44 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-const {hasComment, toW3CTraceContext} = require('./util');
-const {tracer} = require('@opencensus/nodejs');
+const {hasComment} = require('./util');
+const {OpenCensusProvider, OpenTelemetryProvider} = require('./provider')
 
 const defaultFields = {
     'route': true,
     'tracestate': false,
     'traceparent': false,
 };
 
+const providerMap = {
```

Comment:
We shouldn't have to set this up in the package itself. Let's move this into provider.js